plugin/chatgpt.vim

```diff
@@ -13,8 +13,8 @@ import sys
 try:
     import openai
 except ImportError:
-    print("Error: openai module not found. Install with pip.")
-    raise
+    print("Error: OpenAI module not found. Please install with Pip and ensure equality of the versions given by :!python3 -V, and :python3 import sys; print(sys.version)")
```

Comment:
This seems like a good idea, but we should make sure we lave the `OpenAI` all lower cases to match the module name.

Reply:
Yes, thank you, this was amended.